src/core/gui/PdfFloatingToolbox.h

```diff
@@ -0,0 +1,54 @@
+#pragma once
+
+#include "control/Control.h"
+#include "control/tools/PdfElemSelection.h"
+#include "gui/XournalView.h"
+
+#include "GladeGui.h"
+
+
+class PdfFloatingToolbox {
```

Comment:
It might make sense to have both the floating toolbox tool and the PdfFloatingToolbox share a base class...